packages/terra-menu/src/_MenuContent.jsx

```diff
@@ -350,7 +346,7 @@ class MenuContent extends React.Component {
         role="dialog"
         onKeyDown={this.onKeyDown}
       >
-        <ContentContainer header={header} fill={this.props.isHeightBounded || this.props.index > 0}>
+        <ContentContainer header={header} fill={this.props.isHeightBounded}>
```

Comment:
why was the index prop used here before ?

Reply:
index > 0 would imply that it is sub-menu that is being rendered.

Reply:
so how we are handling this scenario now ? after removing this ?